js/api/geolocation/index.js

```diff
@@ -0,0 +1,26 @@
+// Configuration for geolocation options, using gps, timeout and age
+const GEO_CONFIG = {
+  enableHighAccuracy: true, 
+  timeout: 10000, 
+  maximumAge: 1000,
+};
+
+// Returns the location if not timeout
+export const geolocation = () => {
+  return new Promise((resolve, reject) => {
+    navigator.geolocation.getCurrentPosition( 
+      (position) => { 
+        resolve(position); 
+      }, 
+      (error) => {
+        reject(error);
+      }, 
+      GEO_CONFIG 
```

Comment:
`GEO_CONFIG,` is probably OK with the latest version of Babel if es-2017 is enabled. If not, not an issue.